src/events/guild.rs

```diff
@@ -0,0 +1,43 @@
+use db::sea_orm::{Set, ActiveModelTrait, EntityTrait, ModelTrait};
+use serenity::{
+    model::guild::{
+        GuildUnavailable,
+        Guild
+    }, 
+    client::Context
+};
+
+use crate::Connection;
+
+use super::Handler;
+
+
+impl Handler {
+    pub async fn handle_guild_create(&self, ctx: Context, g: Guild, is_new: bool){
+        if is_new{
+
+            let guild_id = *g.id.as_u64() as i64;
+
+            db::guild::ActiveModel{
+                guild_id: Set(guild_id),
+                is_compliant: Set(false),
```

Comment:
should we have this be default? is there a scenario where a guild should be created with is_compliant set to true?

Reply:
no, guilds will by default be false and a bot leaving and joining a guild is considered opting-out of the disclaimer